README.md

```diff
@@ -9,15 +9,17 @@ This is a collection of custom [`ember-template-lint`](https://github.com/ember-
 | Name                                                                       | Description                                                        |
 | :------------------------------------------------------------------------- | :----------------------------------------------------------------- |
 | [`no-expression-like-strings`](./docs/rules/no-expression-like-strings.md) | Catch strings that you probably meant to be Handlebars expressions |
+| [`no-forbidden-elements`](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-forbidden-elements.md#no-forbidden-elements) | Catch `<b>, <i>` that you probably meant to be `<strong>, <em>` expressions |
 
 ## Configurations
 
 The following sets of rules are available for your `ember-template-lint` configuration to extend from:
 
-| Name                   | Description                                                |
-| :--------------------- | :--------------------------------------------------------- |
-| `avoid-possible-typos` | Rules meant to catch possible typos in your templates      |
-| `base`                 | The base set of rules used across all Movable Ink projects |
+| Name                   | Description                                                          |
+| :--------------------- | :------------------------------------------------------------------- |
+| `avoid-possible-typos` | Rules meant to catch possible typos in your templates                |
+| `avoid-b-and-i-tags`   | Rules meant to catch `<b>` and `<i>` tags, use `<strong>` and `<em>` |
```

Comment:
What do you think about a more general name for this configuration? Since the intention here was to guide devs toward recommended tags, maybe something like `avoid-deprecated-elements`?

This is especially useful if we expand to other tags we want to discourage later on!